app/namespace.go

```diff
@@ -56,6 +57,47 @@ var (
 		"us-east-1",
 		"us-west-2",
 	}
+	sinkNameFlag = &cli.StringFlag{
+		Name:     "sink-name",
+		Usage:    "Provide a name for the export sink",
+		Required: true,
+	}
+	sinkAssumedRoleFlagOptional = &cli.StringFlag{
+		Name:  "role-arn",
+		Usage: "Provide role arn for the IAM Role",
```

Comment:
[Nit] should we mention anything that suggests that the IAM Role has to be in the same account as the S3 bucket? Or is that too verbose for here?

Reply:
Yeah. I think in this step is what user needs to create the role beforehand. So we'll need to provide a documentation on how to set the role up and noted the role need to be on the same account of S3